### DISCUSSION TITLE
invitation to github organization : cannot be invited

### DISCUSSION THREAD

**Discussion by sano-t-bns** [community/community#112198] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I am unable to re-invite a particular user.

When I invite this user with an email address, the message "sorry, XXX cannot be invited to XXX" appears on the screen and the email is not sent.

When I invite a user with a user account, the account is marked with ✓ and I cannot click the "Invite" button.

This user can't access to this organization now.

Only one user is in this situation.

How can I fix this?

**Comment by Shai69bb** [comment]
Need to figure out which user has more privileges and delete the user with less privileges?

**Reply by sano-t-bns** [reply]
Thank you for your reply 🙏 
The problem is that the invitation cannot be sent again and the user is unable to join the organization.
User invitation slots are available.

**Comment by grandmas-favorite** [comment]
Thanks for posting in the GitHub Community, @sano-t-bns !

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Actions category is solely related to conversations around the Actions product. This question should be in the General category. I've gone ahead and moved it for you. Good luck!

**Comment by marallyn** [comment]
Any resolution to this problem? We are experiencing the same issue.
